blogs/2018/09/07/engineering-with-azure-pipelines.md

```diff
@@ -0,0 +1,71 @@
+
+Up until very recently, we relied on the OSS community's default choices for public continuous integration: [Travis CI](https://travis-ci.org/) took care of our Linux and macOS builds, while the Windows builds were under [AppVeyor](https://www.appveyor.com/)'s responsibility. Additionally, we used [Coveralls](https://coveralls.io/) to provide detailed test coverage reports. These services provide quality reports for PRs and code branches on our public repo, as they automate [compilation](https://github.com/Microsoft/vscode/blob/master/build/lib/compilation.ts), run [code hygiene checks](https://github.com/Microsoft/vscode/blob/master/build/gulpfile.hygiene.js) and [execute several test](https://github.com/Microsoft/vscode/tree/master/test) suites, all of which is essential in order to maintain quality in a distributed team with lots of incoming contributions. This combination of services require the understanding and maintenance of at least 3 different systems, each with its own special file formats, syntax, quirks, limitations, etc.
+
+## Adopting Azure Pipelines
+
+Earlier this year, we were reached out by the Azure Pipelines (then Visual Studio Team Services) team to try out something new. This was the outcome:
+
+<br />
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">🎉 Hey, did you know <a href="https://twitter.com/code?ref_src=twsrc%5Etfw">@code</a> now uses shiny new <a href="https://twitter.com/VSTS?ref_src=twsrc%5Etfw">@vsts</a> Public Projects and YAML builds to keep its quality up across all 3 platforms simultaneously?! 🤯 Check out the builds at <a href="https://t.co/Q8kYAB1lKG">https://t.co/Q8kYAB1lKG</a>. More about Public Projects at <a href="https://t.co/o0WoNKA4g1">https://t.co/o0WoNKA4g1</a> cc <a href="https://twitter.com/chrisrpatterson?ref_src=twsrc%5Etfw">@chrisrpatterson</a> <a href="https://t.co/Yyhwg19d9z">pic.twitter.com/Yyhwg19d9z</a></p>&mdash; João Moreno (@joaomoreno) <a href="https://twitter.com/joaomoreno/status/991958010661670912?ref_src=twsrc%5Etfw">May 3, 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<br />
+
+This announcement marks our move to a more streamlined continuous integration solution. Our builds now run simultaneously across all platforms, [check it out](https://aka.ms/vscode-builds):
+
+![VSCode Project Build Page](build.png)
```

Comment:
@joaomoreno can you update the screenshot with the latest (Azure DevOps) branding? Retaking should suffice.

Reply:
done!